### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Spark Local Console – Locally run_Keyboard: When Focus is on “Enable parallel execution” check-box, for 3 tabs focus is not visible before going to “Help” button.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3604]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–New Feature; #MAS2.4.3; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS 2.4.3

**Repro Steps:**
1.Download IntelliJ application from the URL:  https://www.jetbrains.com/idea/download/#section=windows .
2.Launch IntelliJ IDEA.
3.Tab till Create New Project button and press enter
4.Tab till Azure Spark/HDInsight-->Spark Project Scala
5.Tab till Build Tools-->Maven
6.Tab till Next button and press enter
7.Tab till finish button and press enter
8. Navigate to Run --> Edit Configurations-->Apache Spark on HDInsight > [Spark on HDInsight] myApp
9.Tab till Locally run tab and press enter
10.Tab till “Enable parallel execution” check-box.
11.Tab once and verify whether focus is pointing to the next control “Help” button or not.

**Actual Result:**
When Focus is on “Enable parallel execution” check-box, for 3 tabs focus is not visible before going to “Help” button.

**Expected Result:**
When Focus is on “Enable parallel execution” check-box, If we tab focus should immediately point to the “Help” button which is the next control present in the page.

**User Impact:**
Keyboard users will face difficulty in knowing where the focus is pointing, If the focus is not visible for 3 tabs before going to “Help” button.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

[Bug_5_Focus order issue after check-box in Spark console-Locally Run.pptx](https://github.com/microsoft/azure-tools-for-java/files/3645406/Bug_5_Focus.osrder.issue.after.check-box.in.Spark.console-Locally.Run.pptx)

**Comment by t-rufang** [comment]
Not repro on my PC. Need to verify it later.
Note: when foucs is at `Enable parallel execution` checkbox, tab 3 times, focus is not visible. tab again focus will be at help button.
![image](https://user-images.githubusercontent.com/32627233/65659707-1d52ee80-e05f-11e9-9315-9fd4521af000.png)

**Comment by t-rufang** [comment]
@v-lasiva I can't repro this issue on my PC. Could you double check it? If it still repro for you, please ping me on Teams and let's have a quick call.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749706
